internal/config/config.go

```diff
@@ -44,6 +50,15 @@ func loadConfig() *Config {
 	viper.SetDefault(SlackAPIKey, "")
 	viper.BindEnv(SlackAPIKey, "SLACK_API_KEY")
 
+	viper.SetDefault(TwilioAccID, "")
+	viper.BindEnv(TwilioAccID, "TWILIO_ACC_ID")
```

Comment:
Please add these env vars to the helm chart as well. They should be referenced in values.yaml and then added to secret.yaml, and then the chart version can be bumped in Chart.yaml.
This will allow us to use these options immediately in staging/production systems running in Kubernetes.